.make/test.mk

```diff
@@ -187,6 +193,22 @@ test-remote-no-coverage: prebuild-check $(SOURCES)
 test-migration: prebuild-check migration/sqlbindata.go migration/sqlbindata_test.go
 	F8_RESOURCE_DATABASE=1 F8_LOG_LEVEL=$(F8_LOG_LEVEL) go test $(GO_TEST_VERBOSITY_FLAG) github.com/fabric8-services/fabric8-wit/migration
 
+.PHONY: test-e2e
+## Runs the end-to-end tests WITHOUT producing coverage files for each package.
+test-e2e: prebuild-check deps docker-compose-up $(SOURCES)
+	$(call log-info,"Running tests: $@")
+	# ## Clone the fabric8-test repo
+	@if [ "$(FABRIC8_TEST_DIR)" ]; then \
+		echo "NOT Removing existing fabric8-test dir $(FABRIC8_TEST_DIR)"; \
+		rm -rf $(FABRIC8_TEST_DIR); \
+	fi
+	$(GIT_BIN_NAME) clone --depth=1 $(FABRIC8_TEST_REPO)
```

Comment:
Please append `$(FABRIC8_E2E_TEST_DIR)` to this line.

Reply:
Do you mean rename FABRIC8_TEST_DIR to FABRIC8_E2E_TEST_DIR?

Reply:
Done.

Reply:
@rgarg1 Where is this done? Have you updated this branch yet?